config/jobs/kubernetes-csi/external-health-monitor/external-health-monitor-config.yaml

```diff
@@ -23,6 +23,9 @@ presubmits:
         - runner.sh
         args:
         - ./.prow.sh
+        env:
+        - name: CSI_SNAPSHOTTER_VERSION
+          value: "v3.0.2"
```

Comment:
Should this be "master"?

Reply:
This job runs with the Kubernetes and sidecar version that is specified inside the prow.sh job, which shouldn't depend on an unreleased snapshotter. So using the fixed version seems fine here.

One could also argue that this particular job shouldn't override `CSI_SNAPSHOTTER_VERSION`.